src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -236,6 +236,8 @@ java.launcher.cls.error6=\
 java.launcher.cls.error7=\
      Error: Unable to initialize main class {0}\n\
      Caused by: {1}: {2}
+java.launcher.cls.error8=\
+    Error: no non-private no argument constructor found in class {0}
```

Comment:
Since this feature is for new developers then the error messages will need to be understand, maybe it should be "zero argument" and give an example to help.

Reply:
```
java.launcher.cls.error8=\
    Error: no non-private zero argument constructor found in class {0}\n\
    remove private from existing constructor or define as:\n\
\   public {0}() {}
```